docs/man/man3/hal_create_thread.3hal

```diff
@@ -18,23 +18,21 @@ The name of the thread
 The interval, in nanoseconds, between iterations of the thread
 
 .IP \fIuses_fp\fR
-Must be nonzero if a function which uses floating-point will be attached
-to this thread.
+Must be nonzero if a function which uses floating-point will be attached to this thread.
 
 .SH DESCRIPTION
-\fBhal_create_thread\fR establishes a realtime thread that will
-execute one or more HAL functions periodically.
+\fBhal_create_thread\fR establishes a realtime thread that will execute one or more HAL functions periodically.
 
-All thread periods are rounded to integer multiples of the hardware timer
-period, and the timer period is based on the first thread created.  Threads
-must be created in order, from the fastest to the slowest.  HAL assigns
-decreasing priorities to threads that are created later, so creating them
-from fastest to slowest results in rate monotonic priority scheduling.
+All thread periods are rounded to integer multiples of the hardware timer period,
+and the timer period is based on the first thread created.
+Threads must be created in order, from the fastest to the slowest.
+HAL assigns decreasing priorities to threads that are created later,
+so creating them from fastest to slowest results in rate monotonic priority scheduling.
```

Comment:
This hunk looks different from what I expect - usually "semantic line-breaks" put each sentence on its own line.  These sentences are still split across multiple lines.  This pattern happens in a couple of places in this PR.

Or is something else going on here that I don't understand?

Reply:
One sentence per line is the best possible. Here, I was just using the comma as a natural location for a line break, which typically (but not always, depends on the the kind of the comma) is enough for the auto-translators not to mess it up. Had felt a bit bad to have a line that is twice as long as the other lines in this man page.

Reply:
Ok :-)